README.md

```diff
@@ -8,9 +8,10 @@
 
 # ![logo](./docs/images/logo_arlon.svg)
 
-Arlon is a lifecycle management and configuration tool for Kubernetes clusters.
-It allows an administrator to compose, deploy and configure a large number of
-*workload clusters* in a structured, predictable way.
+Arlon is a declarative, gitops based fleet management tool for Kubernetes clusters.
+It allows administrators to deploy, secure and upgrade a large number of
+*workload clusters* in a structured, predictable manner. Arlon makes Kubernetes cluster fleet management secure, version controlled, auditable and easy to perform at scale. 
+
```

Comment:
Lost in this (and the previous version) is the fact that Arlon manages application deployment (via profiles) in addition to cluster / infrastructure management. If you can find a way to incorporate that as well, that'd be great. In fact, when managing external clusters, Arlon doesn't manage infrastructure at all: only apps and configurations that go in the clusters.